src/review/const.py

```diff
@@ -10,3 +10,12 @@ class EditorialDecisions(EnumContains):
     UNDECLINE = 'undecline'
     MINOR_REVISIONS = 'minor_revisions'
     MAJOR_REVISIONS = 'major_revisions'
+
+
+class ReviewerDecisions(EnumContains):
+    DECISION_ACCEPT = 'accept'
+    DECISION_MINOR = 'minor_revisions'
+    DECISION_MAJOR = 'major_revisions'
+    DECISION_REJECT = 'reject'
+    DECISION_NO_RECOMMENDATION = 'none'
```

Comment:
This is just a Q, why do we need a `'none'` value as a choice as opposed to leaving the field empty (`None`)